docs/search/search_engines/elastic_search/elastic_search_overview.md

```diff
@@ -19,4 +19,22 @@ Start with the [Set up Elasticsearch](https://www.elastic.co/guide/en/elasticsea
 
 **Prerequisite**
 
-To proceed you need to be familiar with how indexing, filtering and queries work.
\ No newline at end of file
+To proceed you need to be familiar with how indexing, filtering and queries work.
+
+## Update Elasticsearch schema
+
+Whenever you make any changes in case of variables (for example, environmental ones) or configuration files, you need to erase Elasticsearch index, update the schema, and rebuild the index.
```

Comment:
I'm not sure about this condition. We do it here because something changes on kernel side, right?

Reply:
We do it in #2341 because kernel changes.
We do it here because we change the config.